internal/app/table_failed.go

```diff
@@ -2,12 +2,13 @@ package app
 
 import (
 	"fmt"
+	"os"
 	"sort"
 	"strings"
 
 	"github.com/charmbracelet/lipgloss"
-
 	"github.com/mfridman/tparse/parse"
+	"golang.org/x/term"
 )
```

Comment:
The classic blocks for ordering are

- std lib
- new line
- external dependencies
- new line
- subpackages of current module

So here, it should be

```go
 import ( 
          "fmt" 
          "os" 
          "sort" 
          "strings" 
 
          "github.com/charmbracelet/lipgloss" 
          "golang.org/x/term"

          "github.com/mfridman/tparse/parse" 
  )
```

Reply:
Fixed by ce79ac6ade070362a86b6e7bbab8a544928135f2